### ISSUE TITLE
'New project' option for current video to clear segments but not overwrite existing CSV

### ISSUE THREAD

**Issue by Wiethoofd** [mifi/lossless-cut#557]
Currently the easiest way to start a 'new project', while maintaining previously created segments for a video is to rename the existing .csv with those segments and reload the video file, alternatively you 'Save project' and remove all segments.

An option to start a new/clean project in a new file for an alternative cut of your timeline (if it is more than 1 segment that needs to be exported) would be appreciated. Perhaps a way to swap quickly between one or more projects for the same video file in the UI could be useful too if it detects multiple .csv with the `llc-edl` suffix.

**Comment by markusd1984** [comment]
Similar to #537 option to Clear all Segments, which should allow to save as a new project file, and thus reinforce the value as well of having the video source file written in the project file so it can be automatically loaded as well.

**Comment by mifi** [comment]
This can be achieved by
1. Make a copy of your auto generated csv OR File -> save CSV
2. Edit -> Segments -> Clear all segments

Is that good enough?

**Comment by markusd1984** [comment]
@mifi indeed it's a workaround, thanks didn't notice the clear all under edit > segments, probably sufficient as this is likely not such regular task.

But an ability to have new duplicated project files for purpose of keeping a clone or even maintain multiple project versions really requires the **CSV to be linked to the source file**  as well, not just the other way around, that would be ideal, which goes back to 
https://github.com/mifi/lossless-cut/issues/545#issue-762438966 which I think we haven't picked up yet as a feature request?

Just like traditional editors and most apps do, opening a project file and not the media first, easier for versioning rather than duplicating both source and project files, renaming manually in file explorer and then opening the video file. Thus it saves time, space (no duplicate source files only multiple project files) and easier. :blush:

**Comment by mifi** [comment]
I agree that it can be a bit confusing this project file logic. I think duplicatinb source file is not needed if you have multiple csv, then File -> Load Project (CSV), and the project you want to load. But I'm not sure how many people use losslesscut for creating a lot of different projects for a particular file.

**Comment by markusd1984** [comment]
Yes mainly convenience to load various project versions/states with one click as supposed to two clicks. 😄 Perhaps generally not that frequent except for very long videos 2-3hrs+ and those that may go through review/feedback process with other stakeholders involved which may result in re-opening the project at a later day, where then also maintaining different project versions can be beneficial to keep track of changes.

**Comment by mifi** [comment]
As of now, it's possible to open .llc project files and it will open the media file referenced from within. But I still want losslesscut to "just work" when it opens a media file alone, and auto create a project file without bothering the user. so I'm not sure which actionable improvements can be made from now on
